terraform/hydrocron-sns.tf

```diff
@@ -0,0 +1,16 @@
+# SNS topic for CNM responses
+resource "aws_sns_topic" "hydrocron_sns_topic_cnm_response" {
+  name         = "${local.aws_resource_prefix}-cnm-response"
+}
+
+
+resource "aws_sns_topic_policy" "hydrocron_sns_topic_cnm_response_policy" {
+  arn = aws_sns_topic.hydrocron_sns_topic_cnm_response.arn
+  policy = data.aws_iam_policy_document.sns-resource-policy.json
+}
+
+resource "aws_sns_topic_subscription" "hydrocron_cnm_sqs_target" {
```

Comment:
I don't think this is applicable to SQS but there is a [Terraform argument](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic_subscription#endpoint_auto_confirms) for `endpoint_auto_confirms`.  But it looks like this might be more for HTTP endpoints. There is also the [AWS CLI command](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sns/confirm-subscription.html): `sns confirm-subscription`. Not sure either of those will be helpful though.